lms/djangoapps/support/serializers.py

```diff
@@ -86,3 +86,16 @@ def serialize_user_info(user, user_social_auths=None):
                 'uid': user_social_auth.uid,
             })
     return user_info
+
+def serialize_sso_records(user_social_auths):
+    """
+    Serialize user social auth model object
+    """
+    sso_records = []
+    for user_social_auth in user_social_auths:
+        sso_records.append({
+            'provider': user_social_auth.provider,
+            'uid': user_social_auth.uid,
```

Comment:
It's also good to serialize the `created` and `modified` dates

Reply:
Also, because the data in the column `extra_data` sometimes is useful, why not also serialize those?